mtp_noms_ops/settings/docker.py

```diff
@@ -4,13 +4,10 @@
 from mtp_common.stack import get_current_pod
 
 from .base import *  # noqa
-from .base import ENVIRONMENT, os
+from .base import DEBUG, ENVIRONMENT, SECRET_KEY, os
```

Comment:
Given we have `from .base import *` above, is this import of `DEBUG` and `SECRET_KEY` for the benefit of your editor or am I missing something here?

Reply:
code inspection tools don't seem to like variables that come from the `*` import 🤷